models/articles.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection.js");
+
+exports.selectArticlesById = (article_id) => {
+  if (isNaN(article_id)) {
+    return Promise.reject({ status: 400, message: "invalid request" });
+  }
```

Comment:
Nothing wrong with handling it this way. But PSQL will throw it's own error as article_id is expected to be an INT, you could handle this error in a PSQLhandler, checking err.code === "23P02" or whatever it is :)